mbrng/mb_mmd_subs.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 
 #
-# Generated Sun Oct 15 16:24:42 2017 by generateDS.py version 2.28b.
-# Python 2.7.13 (default, Jan 19 2017, 14:48:08)  [GCC 6.3.0 20170118]
+# Generated Tue Mar 12 17:44:10 2019 by generateDS.py version 2.30.11.
+# Python 3.7.2 (default, Jan 10 2019, 23:51:51)  [GCC 8.2.1 20181127]
```

Comment:
It probably doesn't make a difference for the generated code, but the dockerfiles in the https://github.com/metabrainz/sir repo all still reference Python 2.7.

Reply:
Regenerated using Python 2.7 at https://github.com/metabrainz/mb-rngpy/pull/3/commits/637cdde319052a337e8c7e9c7a0b662118729830#diff-912a32f8a1dd3fb9cbb549b265bf754cR5